Helper/Data.php

```diff
@@ -346,11 +414,17 @@ public function cronSubscribeAll($list)
      */
     public function cronAbandonedcart($orders)
     {
-        // Get sync interval and fields from settings.
-        $syncTime = str_replace(':', ' ', $this->getGeneralConfig('syncTime'));
-        $fields = explode(',', $this->getGeneralConfig('productfields'));
         $currentDate = strtotime(date('Y-m-d H:i') . ':00');
         foreach ($orders as $row) {
+            $websiteId = $this->getWebsiteIdByStoreID($row['store_id']);
+            if (! $this->isAbandonedCartEnabledForWebsite($websiteId)) {
```

Comment:
Shouldn't it also check if extension is enabled?

Reply:
Yes, forgot enable checks. Added